components/Pages/DocuSignPage/CartTotal.tsx

```diff
@@ -32,59 +32,51 @@ const CartTotal = () => {
           </h2>
           <hr className="border border-grey -mt-1" />
         </div>
-        <div className="flex flex-col gap-2 text-left font-share place-self-center">
+        <div className="flex flex-col gap-2 text-left font-share place-self-start">
           <div className="w-full flex justify-between gap-2 md:text-xl">
             <p>1x</p>
-            <p>Automized Contract</p>
+            <p className="md:text-[24px]">Automized Contract</p>
           </div>
         </div>
 
         <div className="flex flex-col gap-2 font-share text-[11px]">
           <div className="w-full flex items-center gap-2 ">
-            <p>Sub-Total</p>
+            <p className="md:text-[18px]">Sub-Total</p>
             <MoneyPara>2.00</MoneyPara>
           </div>
           <div className="w-full items-center flex gap-2">
-            <p>Taxes</p>
+            <p className="md:text-[18px]">Taxes</p>
             <MoneyPara>0.34</MoneyPara>
           </div>
           <hr className="border border-grey w-full" />
           <div className="ml-auto">
-            <div>
-              <MoneyPara>2.14</MoneyPara>
-            </div>
+            <MoneyPara>2.14</MoneyPara>
           </div>
+          <ReadHereLink
+            label="Go back to download for free"
+            className="text-grey pt-0 text-[15px] hidden md:block"
+            open={() => router.push('/contract-builder')}
+          />
         </div>
       </div>
-      <div className="flex flex-col gap-8 items-center md:items-start text-center">
+      <div className="flex flex-col gap-4 items-center md:items-start text-center">
         <Button
-          className="w-4/5 self-center md:self-start md:w-fit p-4"
+          className="w-[95%] md:w-fit self-center p-4 md:px-14 md:py-6 md:text-[20px]"
           onClick={() => null}
```

Comment:
onClick={() => null} in the Button component doesn’t serve a purpose. It would be helpful to either connect it to a relevant function or remove it to avoid any confusion.

Reply:
I agree with Zain. Why is the onClick a null function?